README.md

```diff
@@ -98,3 +98,35 @@ remove the .spec dictionary from the yaml)
 
 NOTE: Deploying from the catalog will only work in the opendatahub namespace
 since the operator will already be deployed
+
+## Sample Demo Deployments
+
+### OptaPlanner Demo
+To demonstrate OptaPlanner with the dashboard and admin screens with a smaller resource requirement, we can skip several portions of the demo.  Using the environment file similar to the following:
+```.env
+# oc_login
+OC=~/bin/oc
+OC_URL=https://openshift-host:8443
+OC_USER=admin
+OC_PASSWORD=admin
+
+# Datagrid
+DATAGRID_INSTANCES=1
+DATAGRID_CPU=1000m
+DATAGRID_MEMORY=2048Mi
+
+# OptaPlanner
+OPTAPLANNER_REQUEST_CPU=1000m
+OPTAPLANNER_REQUEST_MEMORY=2000Mi
+OPTAPLANNER_LIMIT_CPU=2000m
+OPTAPLANNER_LIMIT_MEMORY=4000Mi
+
+# Frontend
+FRONTEND_MINI=true
+FRONTEND_SKIP_MOBILE=true
+```
+
+Then execute the make instructions ton install the required components (datagrid, optaplanner, and frontend):
+```bash
+make datagrid optaplanner frontend
+```
```

Comment:
Could you please also add instructions on how to use the demo? Similar to step 8 [here](https://github.com/rhdemo/2019-demo4-optaplanner#running-mechanical-scheduling-demo-only-optaplanner-infinispan-webgame-server--dashboard-ui). Motivation: this will be used by people who aren't familiar with the whole demo and only interested in running a simulation for the scheduling part.

Reply:
added